src/taipy/gui/_gui_cli.py

```diff
@@ -23,6 +23,7 @@ def _create_parser(cls):
         gui_parser.add_argument("-H", "--host", nargs="?", default="", const="", help="Specify server host")
 
         gui_parser.add_argument("--ngrok-token", nargs="?", default="", const="", help="Specify NGROK Authtoken")
+        gui_parser.add_argument("--webapp-path", nargs="?", default="", const="", help="Specify webapp path")
```

Comment:
"Specify webapp path" -> "The path to the WebApp to be used. The default is the webapp directory under gui in the Taipy GUI package directory."